Source/WebKit/UIProcess/RemoteLayerTree/ios/RemoteScrollingCoordinatorProxyIOS.mm

```diff
@@ -161,8 +162,23 @@
                 scrollingStateNode.setFooterLayer(layerTreeHost.layerForID(PlatformLayerID { scrollingStateNode.footerLayer() }));
             break;
         }
+        case ScrollingNodeType::PluginScrolling: {
+            ScrollingStatePluginScrollingNode& scrollingStateNode = downcast<ScrollingStatePluginScrollingNode>(currNode);
+
+            if (scrollingStateNode.hasChangedProperty(ScrollingStateNode::Property::ScrollContainerLayer)) {
+                auto platformLayerID = PlatformLayerID { scrollingStateNode.scrollContainerLayer() };
+                auto remoteLayerTreeNode = layerTreeHost.nodeForID(platformLayerID);
+                if (remoteLayerTreeNode)
+                    scrollingStateNode.setScrollContainerLayer(remoteLayerTreeNode->layer());
+            }
+
+            if (scrollingStateNode.hasChangedProperty(ScrollingStateNode::Property::ScrolledContentsLayer))
```

Comment:
This makes me think we need a test where the scrolled contents layer changes from non-tiled to tiled (maybe we can only do this with incremental loading).

Reply:
Do not think this is actually a thing that happens.